scripts/update_json_docs.sh

```diff
@@ -23,25 +29,33 @@ function pushDocs () {
     echo "Nothing to commit."
   fi
   cd ..
+
+  echo "Done deploying JSON documentation."
 }
 
 function cleanSubmodule () {
-    echo "Cleaning up..."
+    echo "Cleaning up!"
     git submodule deinit -f ghpages
     git reset HEAD .gitmodules
     git reset HEAD ghpages
     rm -rf ghpages
     rm -f .gitmodules
     rm -rf .git/modules/ghpages
+    echo "Done cleaning up!"
 }
 
+# Run this to verifiy that the docs build successfully.
+buildDocs $TRAVIS_BRANCH
+
 if [ "${TRAVIS_BRANCH}" == "master" ] && [ "${TRAVIS_PULL_REQUEST}" == "false" ]; then
+    buildDocs $TRAVIS_BRANCH
   pushDocs $TRAVIS_BRANCH
   cleanSubmodule
   echo "Done pushing docsite. See: https://googlecloudplatform.github.io/gcloud-python/"
 fi
 
 if [[ ! -z $TRAVIS_TAG ]]; then
+  buildDocs $TRAVIS_TAG
```

Comment:
Do the conditionals ensure that this won't occur if `buildDocs` already had? Or if they allow two calls to the same function, is that what you intended?